src/ext/legit/peek-legit.lisp

```diff
@@ -320,7 +320,13 @@ Notes:
 (define-command peek-legit-select () ()
   (alexandria:when-let ((file (get-matched-file)))
     (quit)
-    (alexandria:when-let ((buffer (find-file-buffer file)))
+    (alexandria:when-let
+        ((buffer (or (and (uiop:file-exists-p file)
+                          (find-file-buffer file))
+                     (find-file-buffer
+                      (format nil "~a~a"
+                              (lem-core/commands/project:find-root (buffer-filename))
+                              file)))))
```

Comment:
Using merge-pathnames seems to be a good idea.

Reply:
Added :+1: